lib/tasks/inventory.rake

```diff
@@ -20,6 +20,32 @@ end
 
 namespace :atc do
   namespace :inventory do
+
+    desc 'Updates the path of SourceObjects via old_path and new_path values in the given path_update_csv_file'
+    task update_source_object_path: :environment do
+      unless ENV['path_update_csv_file']
+        puts 'Please supply a path_update_csv_file'
+        next
+      end
+
+      CSV.foreach(ENV['path_update_csv_file'], headers: true).each do |row|
+        old_path = row['old_path']
+        new_path = row['new_path']
+
+        source_object = SourceObject.for_path(old_path)
+        print "Updating SourceObject #{source_object.id}..."
+        raise "Could not find readable file at: #{new_path}" unless File.readable?(new_path)
```

Comment:
Line 37 is fine where it is, but it can also be moved to line 34, right after new_path is initialized.

Reply:
Just pushed a commit with that update.